RockWeb/Blocks/Crm/BulkUpdate.ascx.cs

```diff
@@ -1297,11 +1339,32 @@ protected void btnConfirm_Click( object sender, EventArgs e )
                 }
                 #endregion
 
-                    pnlEntry.Visible = false;
+                #region workflow
+                var personsService = new PersonAliasService( rockContext );
+                var persons = personsService.Queryable()
```

Comment:
This looks like its selecting a list of PersonAlias records (vs Person records). You can just use the existing `people` list that is created earlier in this method.